libtsuba/src/RDGPartHeader.cpp

```diff
@@ -55,8 +55,29 @@ CopyProperty(
 
 }  // namespace
 
+// TODO(vkarthik): repetitive code from RDGManifest, try to unify
+namespace {
+
+katana::Result<uint64_t>
+Parse(const std::string& str) {
+  uint64_t val = strtoul(str.c_str(), nullptr, 10);
+  if (errno == ERANGE) {
+    return KATANA_ERROR(
+        katana::ResultErrno(), "manifest file found with out of range version");
+  }
+  return val;
+}
+
+const int PARTITION_MATCH_HOST_INDEX = 3;
```

Comment:
I think `kPartitionFile` should be local to the file right? I thought I defined it in the private section no?

Reply:
kPartitionFile is currently a private static member.

One question to ask is if `kPartitionMatchHostIndex` and `kPartitionFile` are so closely related to each other, why are they declared differently?

Reply:
I think I was just following the paradigm for `RDGManifest.h`. I guess it doesn't need to be a private static member. It can just be declared for when it's needed in `RDGPartHeader`.